src/autogluon_dashboard/dashboard.py

```diff
@@ -24,26 +23,103 @@ def upload_to_s3(s3_client: botocore.client, file_name: str, object_name: str, a
         Name of object to store file contents in S3 bucket
     """
     with open(file_name, "rb") as f:
-        s3_client.upload_fileobj(f, BUCKET, object_name, ExtraArgs=args)
+        s3_client.upload_fileobj(f, bucket_name, object_name, ExtraArgs=args)
+
+
+def get_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--per_dataset_csv",
+        type=str,
+        required=True,
+        help="Location of csv file in local filesystem to upload to S3 bucket. Example: sub_folder/file_name.csv",
+    )
+    parser.add_argument(
+        "--all_dataset_csv",
+        type=str,
+        required=True,
+        help="Location of csv file in local filesystem to upload to S3 bucket. Example: sub_folder/file_name.csv",
+    )
+    parser.add_argument(
+        "--per_dataset_s3",
+        type=str,
+        help="Location in S3 bucket to save csv file. Example: sub_folder/file_name.csv",
+        default=CSV_FILES_DIR + "all_data.csv",  # TODO: Add commit ID (if applicable) to S3 file path
+    )
+    parser.add_argument(
+        "--all_dataset_s3",
+        type=str,
+        help="Location in S3 bucket to save csv file. Example: sub_folder/file_name.csv",
+        default=CSV_FILES_DIR + "autogluon.csv",  # TODO: Add commit ID (if applicable) to S3 file path
+    )
```

Comment:
Do we want the user to provide name of the csv file as well or just the sub_folder? Or are we handling user providing a path without the filename correctly?

Reply:
OK. So this one actually kind of conflict with `s3_prefix` you have later.
Let's make it that we only accept s3_prefix, and just save the csv files under the prefix with `utils/xxx.csv` or `dev_data/xxx.csv` like you doing right now.
Users don't need to know where the utils are stored as they should be transparent to the user